apertium-eng.eng.dix

```diff
@@ -27345,17 +27348,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="BS">              <i>BS</i><par n="BBC__n"/></e>
 <e lm="BTW">             <i>BTW</i><par n="BBC__n"/></e>
 <e lm="CAT">             <i>CAT</i><par n="BBC__n"/></e>
-<e lm="CCCP">            <i>CCCP</i><par n="BBC__n"/></e>
+<e lm="CAT">             <i>CAT</i><par n="BBC__n"/></e>
```

Comment:
CAT is already there

Reply:
Also, why did you remove CCCP?

Reply:
That was an accident I replaced CCCP with CAT.